bb_server/src/command/sender.rs

```diff
@@ -14,11 +14,6 @@ pub enum ErrorFormat {
 /// Anyone who can send commands. This could be the server console, a player, a
 /// command block, a panda plugin, etc.
 pub trait CommandSender {
-  /// If this command sender has a position in the world, it should be returned.
-  /// If it does not have a position, None should be returned, and relative
-  /// coordinates will not be available to this sender.
-  fn block_pos(&self) -> Option<Pos>;
```

Comment:
there can be other types of command senders (for example, `/execute at <pos> <command`), which have a block pos, but not a player. this function should stay as-is.

Reply:
done